packages/tooling/fast-tooling-wasm/permutator/__tests__/test.c

```diff
@@ -31,11 +96,221 @@ char *test_dlopen()
     return NULL;
 }
 
+char *test_number_type()
```

Comment:
I think it would be nice to see more comprehensive code comments given this is C and doesn't read as well in code as some other languages may.